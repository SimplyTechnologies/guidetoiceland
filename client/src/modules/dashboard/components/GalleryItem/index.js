import GallerItem from './GalleryItem';

export default GallerItem;