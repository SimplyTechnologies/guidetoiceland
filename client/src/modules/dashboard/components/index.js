export GalleryItem from "./GalleryItem/GalleryItem";
export Gallery from "./Gallery/Gallery";
export FilterButtons from "./FilterButtons";