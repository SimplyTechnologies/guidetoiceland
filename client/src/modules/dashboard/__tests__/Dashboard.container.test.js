import React from 'react';

import { shallow, mount } from 'enzyme';
import renderer from 'react-test-renderer';
import Dashboard from '../Dashboard.container';
import { FilterButtons, Gallery } from '../components'
describe('Dashboard.container', () => {
	let instance;
	let rendered;
	
	beforeEach(() => {
		const component = <Dashboard />
		instance = mount(component);
		rendered = renderer.create(component);
	});

	it('should render correctly', () => {
		expect(rendered).toMatchSnapshot();
	});
	it('should container FilterButons and Gallery components', () => {
		expect(instance.find(FilterButtons).length).toBe(1);
		expect(instance.find(Gallery).length).toBe(1);
	})
});