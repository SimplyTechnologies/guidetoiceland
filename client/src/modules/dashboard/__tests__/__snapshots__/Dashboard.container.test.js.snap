// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard.container should render correctly 1`] = `
<div>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="filter-buttons-col col"
        size={4}
      >
        <button
          className="button selected-button btn btn-secondary"
          onClick={[Function]}
          type="button"
        >
          Popular tips
        </button>
      </div>
      <div
        className="filter-buttons-col col"
        size={4}
      >
        <button
          className="button unselected-button btn btn-secondary"
          onClick={[Function]}
          type="button"
        >
          Verified locals
        </button>
      </div>
      <div
        className="filter-buttons-col col"
        size={4}
      >
        <button
          className="button unselected-button btn btn-secondary"
          onClick={[Function]}
          type="button"
        >
          Latest tips
        </button>
      </div>
      <div
        className="filter-buttons-col col"
        size={4}
      >
        <button
          className="button unselected-button btn btn-secondary"
          onClick={[Function]}
          type="button"
        >
          Newest locals
        </button>
      </div>
    </div>
  </div>
  <div
    className="row"
    style={
      Object {
        "justifyContent": "space-between",
      }
    }
  >
    <div
      className="col animated fadeInUp gallery-col col-12 col-sm-6 col-md-4 col-lg-2"
      style={
        Object {
          "background": "url(https://guidetoiceland.imgix.net/5753/x/0/62461501e91513e89581980bb7b06bf7-.jpg?auto=compress%2Cformat&ch=Width%2CDPR&dpr=1&h=600&ixlib=php-1.1.0&w=300&s=b8d5aefbd39c6a3261a7906bf37086f6) center center no-repeat",
          "backgroundSize": "cover",
          "height": "300px",
        }
      }
    >
      <div
        className="gallery-item-title"
      >
        <span>
          Lorem ipsum ahmist ahad
        </span>
      </div>
      <div
        className="gallery-item-description"
      >
        <span>
          Iceland is spectacular in so many ways and Icelandic nature is quite unique with its vast landscape,  volcanic activity, geothermal areas, glacier lagoons and sceneries, black sand beaches and spectacular mountain ridges.  The country's most recognizable series of attractions though are maybe the endless amount of waterfalls making Iceland quite unique. Once I heard the number 10.000. 
        </span>
      </div>
    </div>
    <div
      className="col animated fadeInUp gallery-col col-12 col-sm-6 col-md-4 col-lg-2"
      style={
        Object {
          "background": "url(https://guidetoiceland.imgix.net/267234/x/0/agust-2014-austfirdir-019.jpg?auto=compress%2Cformat&ch=Width%2CDPR&dpr=1&h=600&ixlib=php-1.1.0&w=300&s=aff79094054018fceff66d7684f6deed) center center no-repeat",
          "backgroundSize": "cover",
          "height": "300px",
        }
      }
    >
      <div
        className="gallery-item-title"
      >
        <span>
          Lorem ipsum ahmist ahad
        </span>
      </div>
      <div
        className="gallery-item-description"
      >
        <span>
          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer.
        </span>
      </div>
    </div>
    <div
      className="col animated fadeInUp gallery-col col-12 col-sm-6 col-md-4 col-lg-2"
      style={
        Object {
          "background": "url(https://guidetoiceland.imgix.net/5538/x/0/69263b425cbe576ed488ffa9ced7e522-.jpg?auto=compress%2Cformat&ch=Width%2CDPR&dpr=1&h=600&ixlib=php-1.1.0&w=300&s=3d56f0abafbbb08150348108b7997195) center center no-repeat",
          "backgroundSize": "cover",
          "height": "300px",
        }
      }
    >
      <div
        className="gallery-item-title"
      >
        <span>
          Lorem ipsum ahmist ahad
        </span>
      </div>
      <div
        className="gallery-item-description"
      >
        <span>
          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer.
        </span>
      </div>
    </div>
    <div
      className="col animated fadeInUp gallery-col col-12 col-sm-6 col-md-4 col-lg-2"
      style={
        Object {
          "background": "url(https://guidetoiceland.imgix.net/9658/x/0/981407afbefdcd03617f300c9661daa4-.jpg?auto=compress%2Cformat&ch=Width%2CDPR&dpr=1&h=600&ixlib=php-1.1.0&w=300&s=026862bb140c41e2dec19c460d6ac25a) center center no-repeat",
          "backgroundSize": "cover",
          "height": "300px",
        }
      }
    >
      <div
        className="gallery-item-title"
      >
        <span>
          Lorem ipsum ahmist ahad
        </span>
      </div>
      <div
        className="gallery-item-description"
      >
        <span>
          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer.
        </span>
      </div>
    </div>
    <div
      className="col animated fadeInUp gallery-col col-12 col-sm-6 col-md-4 col-lg-2"
      style={
        Object {
          "background": "url(https://guidetoiceland.imgix.net/196386/x/0/rsz-img-3054-2.jpg?auto=compress%2Cformat&ch=Width%2CDPR&dpr=1&h=600&ixlib=php-1.1.0&w=300&s=745cbd78701713d013023cd1b08fcfb4) center center no-repeat",
          "backgroundSize": "cover",
          "height": "300px",
        }
      }
    >
      <div
        className="gallery-item-title"
      >
        <span>
          Lorem ipsum ahmist ahad
        </span>
      </div>
      <div
        className="gallery-item-description"
      >
        <span>
          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer.
        </span>
      </div>
    </div>
    <div
      className="col animated fadeInUp gallery-col col-12 col-sm-6 col-md-4 col-lg-2"
      style={
        Object {
          "background": "url(https://guidetoiceland.imgix.net/9782/x/0/598e91aafca4b7f7ff27179c37bfeb0a-.jpg?auto=compress%2Cformat&ch=Width%2CDPR&dpr=1&h=600&ixlib=php-1.1.0&w=300&s=2f1041c4cd2e5a74078cd3114b09cce7) center center no-repeat",
          "backgroundSize": "cover",
          "height": "300px",
        }
      }
    >
      <div
        className="gallery-item-title"
      >
        <span>
          Lorem ipsum ahmist ahad
        </span>
      </div>
      <div
        className="gallery-item-description"
      >
        <span>
          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer.
        </span>
      </div>
    </div>
    <div
      className="col animated fadeInUp gallery-col col-12 col-sm-6 col-md-4 col-lg-2"
      style={
        Object {
          "background": "url(https://guidetoiceland.imgix.net/5753/x/0/62461501e91513e89581980bb7b06bf7-.jpg?auto=compress%2Cformat&ch=Width%2CDPR&dpr=1&h=600&ixlib=php-1.1.0&w=300&s=b8d5aefbd39c6a3261a7906bf37086f6) center center no-repeat",
          "backgroundSize": "cover",
          "height": "300px",
        }
      }
    >
      <div
        className="gallery-item-title"
      >
        <span>
          Lorem ipsum ahmist ahad
        </span>
      </div>
      <div
        className="gallery-item-description"
      >
        <span>
          Iceland is spectacular in so many ways and Icelandic nature is quite unique with its vast landscape,  volcanic activity, geothermal areas, glacier lagoons and sceneries, black sand beaches and spectacular mountain ridges.  The country's most recognizable series of attractions though are maybe the endless amount of waterfalls making Iceland quite unique. Once I heard the number 10.000. 
        </span>
      </div>
    </div>
    <div
      className="col animated fadeInUp gallery-col col-12 col-sm-6 col-md-4 col-lg-2"
      style={
        Object {
          "background": "url(https://guidetoiceland.imgix.net/267234/x/0/agust-2014-austfirdir-019.jpg?auto=compress%2Cformat&ch=Width%2CDPR&dpr=1&h=600&ixlib=php-1.1.0&w=300&s=aff79094054018fceff66d7684f6deed) center center no-repeat",
          "backgroundSize": "cover",
          "height": "300px",
        }
      }
    >
      <div
        className="gallery-item-title"
      >
        <span>
          Lorem ipsum ahmist ahad
        </span>
      </div>
      <div
        className="gallery-item-description"
      >
        <span>
          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer.
        </span>
      </div>
    </div>
    <div
      className="col animated fadeInUp gallery-col col-12 col-sm-6 col-md-4 col-lg-2"
      style={
        Object {
          "background": "url(https://guidetoiceland.imgix.net/5538/x/0/69263b425cbe576ed488ffa9ced7e522-.jpg?auto=compress%2Cformat&ch=Width%2CDPR&dpr=1&h=600&ixlib=php-1.1.0&w=300&s=3d56f0abafbbb08150348108b7997195) center center no-repeat",
          "backgroundSize": "cover",
          "height": "300px",
        }
      }
    >
      <div
        className="gallery-item-title"
      >
        <span>
          Lorem ipsum ahmist ahad
        </span>
      </div>
      <div
        className="gallery-item-description"
      >
        <span>
          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer.
        </span>
      </div>
    </div>
    <div
      className="col animated fadeInUp gallery-col col-12 col-sm-6 col-md-4 col-lg-2"
      style={
        Object {
          "background": "url(https://guidetoiceland.imgix.net/9658/x/0/981407afbefdcd03617f300c9661daa4-.jpg?auto=compress%2Cformat&ch=Width%2CDPR&dpr=1&h=600&ixlib=php-1.1.0&w=300&s=026862bb140c41e2dec19c460d6ac25a) center center no-repeat",
          "backgroundSize": "cover",
          "height": "300px",
        }
      }
    >
      <div
        className="gallery-item-title"
      >
        <span>
          Lorem ipsum ahmist ahad
        </span>
      </div>
      <div
        className="gallery-item-description"
      >
        <span>
          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer.
        </span>
      </div>
    </div>
    <div
      className="col animated fadeInUp gallery-col col-12 col-sm-6 col-md-4 col-lg-2"
      style={
        Object {
          "background": "url(https://guidetoiceland.imgix.net/196386/x/0/rsz-img-3054-2.jpg?auto=compress%2Cformat&ch=Width%2CDPR&dpr=1&h=600&ixlib=php-1.1.0&w=300&s=745cbd78701713d013023cd1b08fcfb4) center center no-repeat",
          "backgroundSize": "cover",
          "height": "300px",
        }
      }
    >
      <div
        className="gallery-item-title"
      >
        <span>
          Lorem ipsum ahmist ahad
        </span>
      </div>
      <div
        className="gallery-item-description"
      >
        <span>
          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer.
        </span>
      </div>
    </div>
    <div
      className="col animated fadeInUp gallery-col col-12 col-sm-6 col-md-4 col-lg-2"
      style={
        Object {
          "background": "url(https://guidetoiceland.imgix.net/9782/x/0/598e91aafca4b7f7ff27179c37bfeb0a-.jpg?auto=compress%2Cformat&ch=Width%2CDPR&dpr=1&h=600&ixlib=php-1.1.0&w=300&s=2f1041c4cd2e5a74078cd3114b09cce7) center center no-repeat",
          "backgroundSize": "cover",
          "height": "300px",
        }
      }
    >
      <div
        className="gallery-item-title"
      >
        <span>
          Lorem ipsum ahmist ahad
        </span>
      </div>
      <div
        className="gallery-item-description"
      >
        <span>
          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer.
        </span>
      </div>
    </div>
    <div
      className="col animated fadeInUp gallery-col col-12 col-sm-6 col-md-4 col-lg-2"
      style={
        Object {
          "background": "url(https://guidetoiceland.imgix.net/5753/x/0/62461501e91513e89581980bb7b06bf7-.jpg?auto=compress%2Cformat&ch=Width%2CDPR&dpr=1&h=600&ixlib=php-1.1.0&w=300&s=b8d5aefbd39c6a3261a7906bf37086f6) center center no-repeat",
          "backgroundSize": "cover",
          "height": "300px",
        }
      }
    >
      <div
        className="gallery-item-title"
      >
        <span>
          Lorem ipsum ahmist ahad
        </span>
      </div>
      <div
        className="gallery-item-description"
      >
        <span>
          Iceland is spectacular in so many ways and Icelandic nature is quite unique with its vast landscape,  volcanic activity, geothermal areas, glacier lagoons and sceneries, black sand beaches and spectacular mountain ridges.  The country's most recognizable series of attractions though are maybe the endless amount of waterfalls making Iceland quite unique. Once I heard the number 10.000. 
        </span>
      </div>
    </div>
    <div
      className="col animated fadeInUp gallery-col col-12 col-sm-6 col-md-4 col-lg-2"
      style={
        Object {
          "background": "url(https://guidetoiceland.imgix.net/267234/x/0/agust-2014-austfirdir-019.jpg?auto=compress%2Cformat&ch=Width%2CDPR&dpr=1&h=600&ixlib=php-1.1.0&w=300&s=aff79094054018fceff66d7684f6deed) center center no-repeat",
          "backgroundSize": "cover",
          "height": "300px",
        }
      }
    >
      <div
        className="gallery-item-title"
      >
        <span>
          Lorem ipsum ahmist ahad
        </span>
      </div>
      <div
        className="gallery-item-description"
      >
        <span>
          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer.
        </span>
      </div>
    </div>
    <div
      className="col animated fadeInUp gallery-col col-12 col-sm-6 col-md-4 col-lg-2"
      style={
        Object {
          "background": "url(https://guidetoiceland.imgix.net/5538/x/0/69263b425cbe576ed488ffa9ced7e522-.jpg?auto=compress%2Cformat&ch=Width%2CDPR&dpr=1&h=600&ixlib=php-1.1.0&w=300&s=3d56f0abafbbb08150348108b7997195) center center no-repeat",
          "backgroundSize": "cover",
          "height": "300px",
        }
      }
    >
      <div
        className="gallery-item-title"
      >
        <span>
          Lorem ipsum ahmist ahad
        </span>
      </div>
      <div
        className="gallery-item-description"
      >
        <span>
          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer.
        </span>
      </div>
    </div>
    <div
      className="col animated fadeInUp gallery-col col-12 col-sm-6 col-md-4 col-lg-2"
      style={
        Object {
          "background": "url(https://guidetoiceland.imgix.net/9658/x/0/981407afbefdcd03617f300c9661daa4-.jpg?auto=compress%2Cformat&ch=Width%2CDPR&dpr=1&h=600&ixlib=php-1.1.0&w=300&s=026862bb140c41e2dec19c460d6ac25a) center center no-repeat",
          "backgroundSize": "cover",
          "height": "300px",
        }
      }
    >
      <div
        className="gallery-item-title"
      >
        <span>
          Lorem ipsum ahmist ahad
        </span>
      </div>
      <div
        className="gallery-item-description"
      >
        <span>
          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer.
        </span>
      </div>
    </div>
    <div
      className="col animated fadeInUp gallery-col col-12 col-sm-6 col-md-4 col-lg-2"
      style={
        Object {
          "background": "url(https://guidetoiceland.imgix.net/196386/x/0/rsz-img-3054-2.jpg?auto=compress%2Cformat&ch=Width%2CDPR&dpr=1&h=600&ixlib=php-1.1.0&w=300&s=745cbd78701713d013023cd1b08fcfb4) center center no-repeat",
          "backgroundSize": "cover",
          "height": "300px",
        }
      }
    >
      <div
        className="gallery-item-title"
      >
        <span>
          Lorem ipsum ahmist ahad
        </span>
      </div>
      <div
        className="gallery-item-description"
      >
        <span>
          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer.
        </span>
      </div>
    </div>
    <div
      className="col animated fadeInUp gallery-col col-12 col-sm-6 col-md-4 col-lg-2"
      style={
        Object {
          "background": "url(https://guidetoiceland.imgix.net/9782/x/0/598e91aafca4b7f7ff27179c37bfeb0a-.jpg?auto=compress%2Cformat&ch=Width%2CDPR&dpr=1&h=600&ixlib=php-1.1.0&w=300&s=2f1041c4cd2e5a74078cd3114b09cce7) center center no-repeat",
          "backgroundSize": "cover",
          "height": "300px",
        }
      }
    >
      <div
        className="gallery-item-title"
      >
        <span>
          Lorem ipsum ahmist ahad
        </span>
      </div>
      <div
        className="gallery-item-description"
      >
        <span>
          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer.
        </span>
      </div>
    </div>
    <div
      className="col animated fadeInUp gallery-col col-12 col-sm-6 col-md-4 col-lg-2"
      style={
        Object {
          "background": "url(https://guidetoiceland.imgix.net/5753/x/0/62461501e91513e89581980bb7b06bf7-.jpg?auto=compress%2Cformat&ch=Width%2CDPR&dpr=1&h=600&ixlib=php-1.1.0&w=300&s=b8d5aefbd39c6a3261a7906bf37086f6) center center no-repeat",
          "backgroundSize": "cover",
          "height": "300px",
        }
      }
    >
      <div
        className="gallery-item-title"
      >
        <span>
          Lorem ipsum ahmist ahad
        </span>
      </div>
      <div
        className="gallery-item-description"
      >
        <span>
          Iceland is spectacular in so many ways and Icelandic nature is quite unique with its vast landscape,  volcanic activity, geothermal areas, glacier lagoons and sceneries, black sand beaches and spectacular mountain ridges.  The country's most recognizable series of attractions though are maybe the endless amount of waterfalls making Iceland quite unique. Once I heard the number 10.000. 
        </span>
      </div>
    </div>
    <div
      className="col animated fadeInUp gallery-col col-12 col-sm-6 col-md-4 col-lg-2"
      style={
        Object {
          "background": "url(https://guidetoiceland.imgix.net/267234/x/0/agust-2014-austfirdir-019.jpg?auto=compress%2Cformat&ch=Width%2CDPR&dpr=1&h=600&ixlib=php-1.1.0&w=300&s=aff79094054018fceff66d7684f6deed) center center no-repeat",
          "backgroundSize": "cover",
          "height": "300px",
        }
      }
    >
      <div
        className="gallery-item-title"
      >
        <span>
          Lorem ipsum ahmist ahad
        </span>
      </div>
      <div
        className="gallery-item-description"
      >
        <span>
          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer.
        </span>
      </div>
    </div>
    <div
      className="col animated fadeInUp gallery-col col-12 col-sm-6 col-md-4 col-lg-2"
      style={
        Object {
          "background": "url(https://guidetoiceland.imgix.net/5538/x/0/69263b425cbe576ed488ffa9ced7e522-.jpg?auto=compress%2Cformat&ch=Width%2CDPR&dpr=1&h=600&ixlib=php-1.1.0&w=300&s=3d56f0abafbbb08150348108b7997195) center center no-repeat",
          "backgroundSize": "cover",
          "height": "300px",
        }
      }
    >
      <div
        className="gallery-item-title"
      >
        <span>
          Lorem ipsum ahmist ahad
        </span>
      </div>
      <div
        className="gallery-item-description"
      >
        <span>
          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer.
        </span>
      </div>
    </div>
    <div
      className="col animated fadeInUp gallery-col col-12 col-sm-6 col-md-4 col-lg-2"
      style={
        Object {
          "background": "url(https://guidetoiceland.imgix.net/9658/x/0/981407afbefdcd03617f300c9661daa4-.jpg?auto=compress%2Cformat&ch=Width%2CDPR&dpr=1&h=600&ixlib=php-1.1.0&w=300&s=026862bb140c41e2dec19c460d6ac25a) center center no-repeat",
          "backgroundSize": "cover",
          "height": "300px",
        }
      }
    >
      <div
        className="gallery-item-title"
      >
        <span>
          Lorem ipsum ahmist ahad
        </span>
      </div>
      <div
        className="gallery-item-description"
      >
        <span>
          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer.
        </span>
      </div>
    </div>
    <div
      className="col animated fadeInUp gallery-col col-12 col-sm-6 col-md-4 col-lg-2"
      style={
        Object {
          "background": "url(https://guidetoiceland.imgix.net/196386/x/0/rsz-img-3054-2.jpg?auto=compress%2Cformat&ch=Width%2CDPR&dpr=1&h=600&ixlib=php-1.1.0&w=300&s=745cbd78701713d013023cd1b08fcfb4) center center no-repeat",
          "backgroundSize": "cover",
          "height": "300px",
        }
      }
    >
      <div
        className="gallery-item-title"
      >
        <span>
          Lorem ipsum ahmist ahad
        </span>
      </div>
      <div
        className="gallery-item-description"
      >
        <span>
          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer.
        </span>
      </div>
    </div>
    <div
      className="col animated fadeInUp gallery-col col-12 col-sm-6 col-md-4 col-lg-2"
      style={
        Object {
          "background": "url(https://guidetoiceland.imgix.net/9782/x/0/598e91aafca4b7f7ff27179c37bfeb0a-.jpg?auto=compress%2Cformat&ch=Width%2CDPR&dpr=1&h=600&ixlib=php-1.1.0&w=300&s=2f1041c4cd2e5a74078cd3114b09cce7) center center no-repeat",
          "backgroundSize": "cover",
          "height": "300px",
        }
      }
    >
      <div
        className="gallery-item-title"
      >
        <span>
          Lorem ipsum ahmist ahad
        </span>
      </div>
      <div
        className="gallery-item-description"
      >
        <span>
          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer.
        </span>
      </div>
    </div>
  </div>
</div>
`;
